internal/broker/authmodes/consts.go

```diff
@@ -0,0 +1,16 @@
+// Package authmodes lists the authentication modes that providers can support.
+package authmodes
+
+const (
+	// PasswordID is the ID of the password authentication method.
+	PasswordID = "password"
```

Comment:
I think we can drop the ID suffix altogether, wdyt? It would become `authmodes.Password`, which feels quite good IMO.

Reply:
Oh, sure... I kept `ID` since initially I was putting them as part of the `brokers` namespace, but since that didn't work (as per circular references), I've created a new namespace... We can even call it `authmodesids` but what we have should be good enough.